pkg/binlog/filename.go

```diff
@@ -30,14 +31,67 @@ var (
 	ErrInvalidBinlogFilename = errors.New("invalid binlog filename")
 )
 
-// VerifyBinlogFilename verify whether is a valid MySQL/MariaDB binlog filename.
-// valid format is `base + '.' + seq`.
-func VerifyBinlogFilename(filename string) error {
+// Filename represents a binlog filename.
+type Filename struct {
+	BaseName string
+	Seq      string
```

Comment:
why not `float64`?

Reply:
maybe we want to keep the number format like `000001`,
we can add one more float64 or int filed to keep the number value, something like space-time tradeoff?

Reply:
I added another int64 field in https://github.com/pingcap/dm/pull/140/commits/d7f60d5dcf1f9caa3fa89c0ad8bbb6821281bd1e